### PR TITLE
Update README.md with new communication channels section

### PR DESCRIPTION
**Pull request by theadactyl**
Adding announce mailing list and project discord channels

### CODE REVIEW

**Comment by theadactyl** [comment]
@GMNGeoffrey not sure if wrap worked this time. Not sure I figured it out in vscode this time around.

File: README.md [hunk 1]
```diff
@@ -7,9 +7,16 @@ OpenXLA is a community-driven, open source ML compiler ecosystem, using the best
 * **[Governance](governance/GOVERNANCE.md)** - Operating processes for OpenXLA
 * **[Community Meetings](https://github.com/openxla/community/tree/main/meetings)** - Schedule, agenda & minutes for public OpenXLA community meetings
 
+## Communication Channels 
+
+* **[`openxla-discuss` Mailing List](https://groups.google.com/a/openxla.org/g/openxla-discuss)** - Technical and community conversations about the OpenXLA project & its components, community meeting invitations & agendas. This is our primary channel. 
+* [`openxla-announce` Mailing List](https://groups.google.com/a/openxla.org/g/openxla-announce) - Less frequent announcements from OpenXLA project maintainers, including project news, events and major releases.
+* [OpenXLA Discord](https://discord.gg/NNvMJ2yd7a) - Synchronous chat about top-level
+project updates, and the development of `/xla` and `/stablehlo`.
+* [IREE Discord](https://discord.gg/BexW7bgHqm) - Synchronous chat about the development of `/iree`.
```
Comment (jpienaar) [inline README.md:16]: and `/pjrt` ? (is the link the room name or repo?)

Comment (theadactyl) [reply README.md:16]: I dont think there's a top-level pjrt channel atm, other than the one specifically for IREE.

Comment (jpienaar) [reply README.md:16]: PJRT developers are all there, so as close to a top-level channel as we have externally. But I'm fine postponing too.

**Review by jpienaar — APPROVED** [review]
Looks good overall, thanks - if you could address Geoffrey's comments first.

**Comment by stellaraccident** [comment]
I don't know why this is lingering without update. The value of having something in the readme is higher than waiting for a comment cycle on typography. Landing. Further iteration welcome.

**Comment by GMNGeoffrey** [comment]
> I don't know why this is lingering without update. The value of having something in the readme is higher than waiting for a comment cycle on typography. Landing. Further iteration welcome.

Yeah those were not intended to be blocking comments

**Pull request merged**